example/testapp/apitest/features/basic.feature

```diff
@@ -75,9 +77,6 @@ Feature: Basic Features for lodash-match-pattern
         }
       ]
       """
-
-  Scenario: Users are searched correctly
-    Given the database is populated with sample users
     When the client gets "/users?name=alec"
```

Comment:
I don't think my get request modifies any server state. This get request just searches for users based on various criteria